doc/nrf/glossary.rst

```diff
@@ -0,0 +1,404 @@
+   See `OpenAMP`_
+
+OpenThread
+   A portable and flexible open-source implementation of the Thread networking protocol.
+
+OpenThread Border Router (OTBR)
+   A router that connects a Thread network to other IP-based networks, like Wi-Fi or Ethernet.
+   A Thread network requires a Border Router to connect to other networks.
+
+OSS repository
+   An |NCS| repository that tracks an upstream Open Source Software counterpart that is externally maintained.
+
+Over-the-Air (OTA)
+   Any type of wireless transmission.
+
+Patch
+   See Commit.
+
+Power Amplifier (PA)
+   A device used to increase the transmit power level of a radio signal.
+
+Power Saving Mode (PSM)
+   A feature introduced in 3GPP Release 12 to improve battery life of IoT (Internet of Things) devices by minimizing energy consumption.
+   The device stays dormant during the PSM window.
+
+Printed Circuit Board (PCB)
+   A board that connects electronic components.
+
+Protocol Data Unit (PDU)
+   Information transferred as a single unit between peer entities of a computer network and containing control and address information or data.
+   PDU mode is one of the two ways of sending and receiving SMS messages.
+
+Pull Request
+   A GitHub Pull Request, a set of commits that are sent for code review using GitHub.
+
+Quality of Service (QoS)
+   The measured overall performance of a service, such as a telephony or computer network, or a cloud computing service.
+
+Radio Co-Processor (RCP)
+   A co-processor offloading radio functions from the host processor.
+   In the |NCS| context, it is tipically used in OpenThread platform designs.
+
+Real-time operating system (RTOS)
+   A computing environment that reacts to input within a specific period of time. A real-time deadline can be so small that system reaction appears instantaneous.
+
+Real-Time Transfer (RTT)
+   Proprietary technology for bidirectional communication that supports J-Link devices and ARM-based microcontrollers, developed by SEGGER Microcontroller.
+
+Receive Data (RXD)
+   A signal line in a serial interface that receives data from another device.
+
+Repository
+   A Git repository in its strict sense, the highest granularity allowed by the Git version control system.
+
+Root of Trust (RoT)
+   The property in a secure system that identifies the initial step of a chain of trust.
+
+Sample
+   An implementation example that showcases a single feature or library
+
+Secure Access Port Protection (SECUREAPPROTECT)
+   A register used to prevent read and write access to all secure CPU registers and secure memory-mapped addresses.
+   See Access Port Protection (APPROTECT).
+
+Security Manager Protocol (SMP)
+   A protocol used for pairing and key distribution.
+
+SEGGER Embedded Studio (SES)
+   A cross-platform Integrated Development Environment (IDE) for embedded C/C++ programming with support for Nordic Semiconductor devices, produced by SEGGER Microcontroller.
```

Comment:
Maybe add: "|NCS| uses a custom Nordic Edition of SES."?

Reply:
@greg-fer Are you sure this is actually a custom edition of SES? I went through the installation process in a sandbox, earlier today, and the installer mentioned a custom license for users of Nordic products. It was also offering a free test license for general users and a professional license, if you had one. I believe this is because the actual IDE is the same. We should verify this.

Reply:
It has an NCS section in Options.

Reply:
I'm pretty sure this depends on the "Nordic Semiconductor nRF CPU Support Package" installed as an additional package. I just managed to get that same section installing the latest "vanilla" SEGGER plus that package. Still, the license management section seems to be different. So I'll just mention the Nordic version available as a zip.

Reply:
Fixed